src/index.js

```diff
@@ -59,6 +59,24 @@ const unexpectedEnzyme = {
       }
     );
 
+    childExpect.exportAssertion(
+      '<ReactWrapper> queried for <string> <assertion>',
+      (expect, reactWrapper, query) => {
+        return expect.shift(reactWrapper.find(query));
```

Comment:
If would be nice if it failed in a detailed way if the selector does not match anything. But we can make that improvement later.